src/client/components/Appheader/AppHeader.style.css

```diff
@@ -1,34 +1,25 @@
-/* AppHeader */
+* {
+  margin: 0;
+  padding: 0;
+}
 
-/* BG */
 .AppHeader-div {
-  background-color: #f8f8f8;
-  width: 100%;
   height: 59px;
-  position: absolute;
+  position: relative;
+  background-color: #f8f8f8;
 }
 
-.AppHeader-div img {
+.Appheader-img-container {
   position: absolute;
+  max-width: 59px;
+  max-height: 30px;
   left: 5.8%;
   right: 79.95%;
   top: 23.73%;
   bottom: 25.42%;
```

Comment:
While here, can you remove the absolute positioning and these 4 values? Instead, just use padding.